tools/precompile-css.R

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env Rscript
```

Comment:
This was already moved into yarn_install.R, so there's no need to bring it back.